core/src/main/java/org/apache/accumulo/core/conf/PropertyType.java

```diff
@@ -262,4 +267,46 @@ public boolean apply(final String input) {
 
   }
 
+  public static class PortRange extends Matches {
+
+    private static final Logger log = LoggerFactory.getLogger(PortRange.class);
+
+    public PortRange(final String pattern) {
+      super(pattern);
+    }
+
+    @Override
+    public boolean apply(final String input) {
+      if (super.apply(input)) {
+        try {
+          PortRange.parse(input);
+          return true;
+        } catch (IllegalArgumentException e) {
+          return false;
+        }
+      } else {
+        return false;
+      }
+    }
+
+    public static Pair<Integer,Integer> parse(String portRange) {
+      int idx = portRange.indexOf('-');
+      if (idx != -1) {
+        int low = Integer.parseInt(portRange.substring(0, idx));
+        if (low < 1024) {
```

Comment:
These checks were not there previously. This PR is already closed and the code committed. You will have to open up tickets for these new issues.